app/templates/views/add-service-local.html

```diff
@@ -0,0 +1,38 @@
+{% extends "withoutnav_template.html" %}
+{% from "components/radios.html" import radios %}
+{% from "components/textbox.html" import textbox %}
+{% from "components/page-header.html" import page_header %}
+{% from "components/page-footer.html" import page_footer %}
+{% from "components/form.html" import form_wrapper %}
+
+{% block per_page_title %}
+    About your service
+{% endblock %}
+
+{% block maincolumn_content %}
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+
+    {{ page_header('About your service') }}
+
+    <p class="govuk-body">Give your service a name that tells users what your messages are about, as well as who they’re from. For example:</p>
+        <ul class="list-bullet">
+          <li>School admissions - Brighton & Hove City Council</li>
+          <li>Electoral services - London Borough of Bromley</li>
+          <li>Blue Badge - Redcar & Cleveland Borough Council</li>
```

Comment:
We use the name of the user’s organisation here:
```suggestion
          <li>School admissions - {{ current_user.default_organisation.name }}</li>
          <li>Electoral services - {{ current_user.default_organisation.name }}</li>
          <li>Blue Badge - {{ current_user.default_organisation.name }}</li>
```

Reply:
Nice 👍